doc-template/things-template.md.mustache

```diff
@@ -0,0 +1,65 @@
+{{#*inline "configuration"}}
+| Property         | Value    |
+|------------------|----------|
+| Configuration ID | {{name}} |
+| Data Type        | {{type}} |
+{{~#if options}}{{#if limitToOptions}}{{else}}
+| Range | {{min}} to {{max}} {{#if step}}with step of {{step}}{{/if}}|{{/if}}{{else}}
+| Range | {{min}} to {{max}} {{#if step}}with step of {{step}}{{/if}}|
+{{/if}}
+{{~#if defaultValue}}| Default Value | {{defaultValue}} |
+{{/if}}
+{{~#if options}}{{#options}}| {{#if @first}}Options{{/if}} | {{label}} ({{value}}) |
+{{/options}}{{/if}}
+{{#if readOnly}}
+**NOTE:** This parameter is read-only.{{/if}}
+{{/inline}}
+{{#thing}}# {{label}}
+
+This describes the Z-Wave device **{{property.modelId}}**, manufactured by **{{property.vendor}}** with the thing type UID of ```{{id}}```. 
+
+{{html2md description}}
```

Comment:
Maybe add `**Description:** ` so this doesn't look too out of context?

Reply:
For most devices, this will be quite a long description. Does a heading like this really make sense? I could add a ```## Description``` header if you like - to me it looks ok as is, but happy to differentiate it if you like. Note that this generates a number of additional sub-sections about the zwave device even if it's all in the description field in the XML.

Reply:
Okay, you know best what is what. Let's leave as is